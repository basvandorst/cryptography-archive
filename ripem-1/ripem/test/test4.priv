-----BEGIN PRIVATE KEY-----
User: test4
PublicKeyInfo:
 MIGbMAoGBFUIAQECAgP/A4GMADCBiAKBgGfFnTYplujGSNRMMO/4M75+4UuPbCy3
 NTNl6fGG1CTa5dY9bMr9pXK3Ah+vMKHv/pxDeM0xdpsFYVSPv0eX51J4TlqPFyRC
 H/VlK5RmP5HGF29dQJkep9GqRMnZxop4EAZ/2CGBnx6wUwhJJ6dmWbJmjhZV0uCT
 Qs6omY+8HHm9AgMBAAE=
MD5OfPublicKey: 31A46B9D8395881FD8F271849CA46E93
EncryptedPrivateKeyInfo:
 MIICoDAaBgkqhkiG9w0BBQMwDQQI9A8U5QByEHICAWQEggKA/qEXkRLhS924N9Nl
 DKRqW2hIZE/mVveS9QabYaeDgWHTC/KM9i6C+2W1J/Ufh18KzAx2kB4a2pe1dGYf
 Ipoe7ySBVClpzDJDUBlX4uLflYr6nvkTR4wh/Fu+uaJM2I1PlzMbD3CB+UqDPCTH
 yWzukMP1WLGRty1riGdZD707LTvdXfSrcODOjKF37tGVleU33ldaatAIAEVHAkcI
 pwvwsP70plQq7BYggJ3igvWPknQ3fyWaqfU8WO/79fT+KzTkyJFUjty30pUWRNdx
 RZtgU0AQzIaEgsA2SHRf4DnVsiNFv2gmw0Slf3z8GXhVYPJL5wxamlOxOjF9Pt9n
 u4vsxYJM0ri0yGt7ukmRwZOYN5B58aZvkAzmL7ZQY+p1pM2Zp9wcKH9x/DB2I1ff
 vh/73T8QXgMOAnfPOCAyKJr+CQdURqHTa6MFKZgol7G+slkgSlZ/aD2NZ7vwzktK
 zrfHJauK6UJDajWlfTatMFbThDMyFx4xcWjzErMgU8heTNen2sG+xrc+hy9MFzOD
 un8bcfdXdmbAc4auyAFYrYcc09nAASSz+nA5lr5WyVPc3P2f73q4FR2EhRjQC74p
 bJLv5BLmfrm6Vesgn367XSzi4jRfm7PSyD6P6Zcf1OCAgckcy9TtemmZEvt3Zc5N
 wxKj4enPh+amHVe7qlAZkRl3fCdT8RFJp0fo4rZZfx3Y5Vv5Aw16lc6uO+gcUIif
 X8lcon9WNxYcoCL4saTpkA/x69k5oi/ouokTKCzIyXXE8xMR95NpwmFd7+yt57Ln
 ZSd5Y0r7ofprzCL43/TtlTZJWTJey12qJlUAnmEgFF6A0+PaNoojtwrm5AwQL91H
 RoUo4A==
-----END PRIVATE KEY-----
