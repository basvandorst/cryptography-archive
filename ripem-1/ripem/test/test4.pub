-----BEGIN PUBLIC KEY-----
User: test4
PublicKeyInfo:
 MIGbMAoGBFUIAQECAgP/A4GMADCBiAKBgGfFnTYplujGSNRMMO/4M75+4UuPbCy3
 NTNl6fGG1CTa5dY9bMr9pXK3Ah+vMKHv/pxDeM0xdpsFYVSPv0eX51J4TlqPFyRC
 H/VlK5RmP5HGF29dQJkep9GqRMnZxop4EAZ/2CGBnx6wUwhJJ6dmWbJmjhZV0uCT
 Qs6omY+8HHm9AgMBAAE=
MD5OfPublicKey: 31A46B9D8395881FD8F271849CA46E93
-----END PUBLIC KEY-----
