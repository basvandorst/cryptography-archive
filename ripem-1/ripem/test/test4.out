-----BEGIN PRIVACY-ENHANCED MESSAGE-----
Proc-Type: 2001,ENCRYPTED
DEK-Info: DES-CBC,951591A47BE1797E
Originator-Name: test4
Originator-Key-Asymmetric:
 MIGbMAoGBFUIAQECAgP/A4GMADCBiAKBgGfFnTYplujGSNRMMO/4M75+4UuPbCy3
 NTNl6fGG1CTa5dY9bMr9pXK3Ah+vMKHv/pxDeM0xdpsFYVSPv0eX51J4TlqPFyRC
 H/VlK5RmP5HGF29dQJkep9GqRMnZxop4EAZ/2CGBnx6wUwhJJ6dmWbJmjhZV0uCT
 Qs6omY+8HHm9AgMBAAF=
MIC-Info: RSA-MD5,RSA,
 uB+If+Z8we2DyyGN7j6Xc45gaqtEbYxkr3IgMnUbOKlpc6z6azy2Ilzi2dDfuPea
 Sf+P17SKIhqK9vMYMK76/au4JyBodhgKmts8+cQn4GTiJJ3835/ejk2Ef63nzdvw
 zJ/Eq4CIBaJsymQrMI72ysMtnRQvH+7Lq7GN1nhenh4HXnTKMU/W2g==
Recipient-Name: test4
Key-Info: RSA,
 YDjDBYjB3li+e3nVgQKgllrRYZhW1wo6fSa26iinSQo5+bUK5CBLUOgW1UsNfnvh
 ddx/CPKKXsPGqofvut6RUrTdmGMsPQu/+3PIiMCA5puO1/x+MPeFPONp0cqmoAew
 IUruOjf0YzmOsaDvM029hRc3sP4Q6m1nFL+zHCS+Jn8=

zhURjalzgvjFlYsEc3pvfanPtHbJ6dDJbyiBuJO12A84KZErr3uTVw+DSsBRjx6c
-----END PRIVACY-ENHANCED MESSAGE-----
